File: pythons/rgb.py
import random

R = random.randint(0,255)
print(R)
maxguess=12

for i in range (1,maxguess+1):
    guess = int(input("Masukkan R:"))
    
    if guess == R :
        print("BENAR")
        break
    elif R-guess <= 10:
        print("PANAS")
        print("Banyak percobaan:",i)
    elif 11 <= R-guess <= 30:
        print("HANGAT")
        print("Banyak percobaan:",i)
    elif R-guess >=31:
        print("DINGIN")
        print("Banyak percobaan:",i)
        
    if i == maxguess+1:
        print("KESEMPATAN MENEBAK HABIS")
        break

File: javas/Rgb.java
import java.util.Random;
import java.util.Scanner;

public class Rgb {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        Random random = new Random();

        int r = random.nextInt(256);
        System.out.println(r);

        int maxGuess = 12;

        for (int i = 1; i <= maxGuess; i++) {
            System.out.print("Masukkan R:");
            int guess = sc.nextInt();

            if (guess == r) {
                System.out.println("BENAR");
                break;
            } else if (r - guess <= 10) {
                System.out.println("PANAS");
                System.out.println("Banyak percobaan:" + i);
            } else if (11 <= r - guess && r - guess <= 30) {
                System.out.println("HANGAT");
                System.out.println("Banyak percobaan:" + i);
            } else if (r - guess >= 31) {
                System.out.println("DINGIN");
                System.out.println("Banyak percobaan:" + i);
            }

            if (i == maxGuess) {
                System.out.println("KESEMPATAN MENEBAK HABIS");
                break;
            }
        }

        sc.close();
    }
}
